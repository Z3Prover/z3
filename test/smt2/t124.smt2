
; Copyright (c) 2015 Microsoft Corporation

(set-option :print-success true)

(declare-datatypes (T) ((LispList nil (cons (car T) (cdr LispList)))))

(declare-const x (LispList Int))

(display (cons 10 nil))
