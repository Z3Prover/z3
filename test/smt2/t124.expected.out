success
success
success
(cons 10 nil)
